admin/class-meta-columns.php

```diff
@@ -453,37 +453,57 @@ protected function determine_score_filters( $score_filters ) {
 	/**
 	 * Retrieves the post type from the $_GET variable.
 	 *
-	 * @return string The current post type.
+	 * @return string The sanitized current post type or an empty string when the variable is not set in $_GET.
 	 */
 	public function get_current_post_type() {
-		return filter_input( INPUT_GET, 'post_type' );
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Reason: We are not processing form information.
+		if ( isset( $_GET['post_type'] ) && is_string( $_GET['post_type'] ) ) {
+			// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Reason: We are not processing form information.
+			return sanitize_text_field( wp_unslash( $_GET['post_type'] ) );
+		}
+		return '';
```

Comment:
Now, if `post_type` doesnt exist in the GET object, we're returning an empty string, whereas, we used to return `null`.

Can we _not_ change this behavior so as to not ensure no regressions? (and if we want to be 100% right there, also change the return type of the function.

The same is true for the other touched functions as well

Reply:
Good point! Fixed!